cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/Measurement.java

```diff
@@ -0,0 +1,52 @@
+package io.smartcat.cassandra.diagnostics;
+
+/**
+ * This class represents a module generated measurement.
+ */
+public class Measurement {
+
+    private String name;
+    private Query query;
```

Comment:
Query should not be part of Measurement because it's too specific. For example, it fits SlowQueryModule but not RequestRateModule.
One approach would be to have a key-value map (or something similar) with measurement values (and tags?).